src/Core/Style.js

```diff
@@ -0,0 +1,113 @@
+import { FEATURE_TYPES } from 'Core/Feature';
+
+function rgba2rgb(orig) {
+    if (orig && orig.includes('rgba')) {
+        const rgb = orig.replace(/\s/g, '').match(/(\d+),(\d+),(\d+),(\d\.\d)/i);
```

Comment:
You need to add a wider range for the regex, see here https://developer.mozilla.org/en-US/docs/Web/CSS/color_value#RGB_colors (you're missing the `%` for example.